src/main/java/frc/robot/RobotContainer.java

```diff
@@ -59,6 +60,7 @@ private RobotContainer() {
 
     // SmartDashboard Buttons
     SmartDashboard.putData("Shoot: speed", new Shoot(0.8, m_shooter));
+    SmartDashboard.putData("tankDrive", new tankDrive( m_drivetrain ));
```

Comment:
@LucaPorta17 @neuenfeldttj you'll probably find the issue before you see this comment.
From what I can see, the reason it doesn't work is you didn't set the tankDrive() command to be the default of the drive train (would appear like [so in the code](https://github.com/The-Charge/Plybot2022/pull/2/files?authenticity_token=9%2F8K%2FwKHP4JtmwZ4hfvOmv%2BoVsx5BV3WbYyZcW4wrytFe3x8pQXcbtTfSDcZpBygGoi7dFDLTNqCjsPFYfN5Eg%3D%3D&file-filters%5B%5D=.java#diff-789275cb88dd16cec774f646d9a1711e27f068723fbaa996d4c3c37f9725d525R71))
There's a checkbox in robot builder to set this.